controller/main_controller.php

```diff
@@ -93,7 +93,7 @@ public function display($route)
 	* @param string $route The route name for a page
 	* @return object $entity The entity object
 	* @throws http_exception
-	* @access public
+	* @access protected
```

Comment:
Well then need to delete the @ access annotation in all files (including those in the core of phpBB).